app/uk/gov/hmrc/transitmovements/services/ObjectStoreService.scala

```diff
@@ -62,4 +76,31 @@ class ObjectStoreServiceImpl @Inject() (client: PlayObjectStoreClient) extends O
           case NonFatal(ex) => Left(ObjectStoreError.UnexpectedError(Some(ex)))
         }
     )
+
+  private val dateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd-HHmmss").withZone(ZoneOffset.UTC)
+
+  override def addMessage(
+    movementId: MovementId,
+    messageId: MessageId,
+    source: Source[ByteString, _]
+  )(implicit ec: ExecutionContext, hc: HeaderCarrier): EitherT[Future, ObjectStoreError, ObjectSummaryWithMd5] =
+    EitherT {
+      val formattedDateTime = dateTimeFormatter.format(OffsetDateTime.ofInstant(clock.instant, ZoneOffset.UTC))
+
+      (for {
+        response <- client.putObject(
+          path = Path.Directory(s"movements/${movementId.value}").file(s"${movementId.value}-${messageId.value}-$formattedDateTime.xml"),
+          content = source,
+          owner = "common-transit-conversion-traders"
```

Comment:
type should be ``` common-transit-convention-traders ```

Reply:
use the constant here: https://github.com/hmrc/transit-movements/blob/c9f8490c97a78e484cf01937dd811ecd189e32d3/app/uk/gov/hmrc/transitmovements/models/ObjectStoreURI.scala#L25